src/Microsoft.Performance.SDK.Runtime.Tests/VersionCheckerTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.Testing.SDK;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NuGet.Versioning;
+
+namespace Microsoft.Performance.SDK.Runtime.Tests
+{
+    [TestClass]
+    public class VersionCheckerTests
+    {
+        [TestMethod]
+        public void BaseLineIs109()
+        {
+            var sut = CreateSut();
+
+            Assert.AreEqual(new SemanticVersion(0, 109, 0), sut.LowestSupportedSdk);
+        }
+
+        [TestMethod]
+        public void VersionLessThanBaselineRejected()
+        {
+            var baseline = new SemanticVersion(1, 2, 3);
+            var version = new SemanticVersion(1, 2, 2);
+
+            var sut = CreateSut(baseline: baseline);
+
+            Assert.IsFalse(sut.IsVersionSupported(version));
+        }
+
+        [TestMethod]
+        [DataRow("0.109.0", "0.109.0", "0.109.0", true)]
+
+        // major being zero (0), anything between lowest version
+        // and current version is acceptable.
+        [DataRow("0.112.0", "0.109.0", "0.109.0", true)]
+        [DataRow("0.112.0", "0.109.0", "0.111.0", true)]
+        [DataRow("0.112.0", "0.109.0", "0.112.0", true)]
+        [DataRow("0.112.0", "0.109.0", "0.112.1", false)]
```

Comment:
Do we want this to be `true`.? As long as we agree to not introduce any interface changes between pre-relase patch versions, this should be ok. @trgibeau what do you think?